ports/esp32/usb_serial_jtag.c

```diff
@@ -95,6 +95,10 @@ void usb_serial_jtag_poll_rx(void) {
     }
 }
 
+#ifndef USB_SERIAL_JTAG_PACKET_SZ_BYTES
+#define USB_SERIAL_JTAG_PACKET_SZ_BYTES USB_SERIAL_JTAG_RDWR_BYTE_S
```

Comment:
The value of `USB_SERIAL_JTAG_RDWR_BYTE_S` is 0 which is why you don't get any bytes here.

Try defining to 64 for now, that was the value before the macro was removed.

Reply:
Thanks for that! 
I did scan the 5.3 migration guide and didn't see anything about that but I guess `USB_SERIAL_JTAG_RDWR_BYTE_S` wasn't necessarily ever intended as an end user/application variable.

Reply:
No, talking directly to the usb serial peripheral (rather than via stdio) is a private "low level" interface so no guarantees, I think!